.github/workflows/workflow.yaml

```diff
@@ -0,0 +1,115 @@
+name: Test and deploy
+
+# define when to run the action
+on:
+  - push
+  - pull_request
+
+env:
+  GITHUB_WORKFLOW: true
+
+jobs:
+  test:
+    name: Test
+    runs-on: ubuntu-20.04
+
+    # test matrix
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version:
+          - 3.6
+          - 3.7
+          - 3.8
+          - 3.9
+        django:
+          - 2.2
+          - 3.0
+          - 3.1
+        database:
+          - sqlite
+          - mysql
+          - postgres
+
+    # additional service containers to run
+    services:
+      # postgres service
+      postgres:
+        # docker hub image
+        image: postgres:9.5-alpine
```

Comment:
postgres:10+ubuntu would be great

Reply:
These are the same requirements as in the tox build. Will happily upgrade to newer releases

Reply:
Any specific reasons for ubuntu as alpine images are smaller, lightweight and faster.